.travis.yml

```diff
@@ -1,10 +1,8 @@
 sudo: false
 language: go
-matrix:
-  include:
-    - go: 1.5.4
-      env: GO15VENDOREXPERIMENT=1
-    - go: 1.6.2
+go:
+- 1.5.4
+- 1.6.2
```

Comment:
Just moved it to the global environment section; it doesn't matter to Go 1.6.x and will still be applied to Go 1.5.x. If we didn't move it we'd end up with two blocks labeled matrix with the chances made here to separate out the small and legacy test runs, so we thought we'd just collapse it down to one matrix block by moving that environment variable declaration